### PR TITLE
8080bw.cpp: Change Jatre Specter to licensed

### PR DESCRIPTION
**Pull request by FoxhackDN**
As described in MT 08149, Jatre Specter was licensed by Taito.

### CODE REVIEW

File: src/mame/midw8080/8080bw.cpp [hunk 1]
```diff
@@ -5999,8 +5999,8 @@ GAMEL(1979, supinvsion,  invaders, invaders,  searthin,  invaders_state, empty_i
...
-GAMEL(1979, jspecter,    invaders, invaders,  jspecter,  invaders_state, empty_init,    ROT270, "bootleg (Jatre)",                    "Jatre Specter (set 1)",                                           MACHINE_SUPPORTS_SAVE, layout_invaders )
-GAMEL(1979, jspecter2,   invaders, invaders,  jspecter,  invaders_state, empty_init,    ROT270, "bootleg (Jatre)",                    "Jatre Specter (set 2)",                                           MACHINE_SUPPORTS_SAVE, layout_invaders )
+GAMEL(1979, jspecter,    invaders, invaders,  jspecter,  invaders_state, empty_init,    ROT270, "Taito / Jatre",                    "Jatre Specter (set 1)",                                           MACHINE_SUPPORTS_SAVE, layout_invaders )
+GAMEL(1979, jspecter2,   invaders, invaders,  jspecter,  invaders_state, empty_init,    ROT270, "Taito / Jatre",                    "Jatre Specter (set 2)",                                           MACHINE_SUPPORTS_SAVE, layout_invaders )
```
Comment (cuavas) [inline src/mame/midw8080/8080bw.cpp:6003]: Should it be “Taito (Jatre license)” or maybe even just “Jatre”?

Comment (FoxhackDN) [reply src/mame/midw8080/8080bw.cpp:6003]: I looked at other similar sets and how they were listed - Ultra Invaders has Taito / Konami, Space King has Taito / Leijac, and Space War has Taito / Sanritsu. If you still think it should be changed then lemme know which to use.

**Comment by happppp** [comment]
If it's still clearly derived from Taito's Space Invaders but they did more modifications than just the title and company credit, it's a bit hard to decide it's "Taito (CompanyB license)". IIRC that's why I picked "Taito / CompanyB" for those.

**Comment by FoxhackDN** [comment]
> If it's still clearly derived from Taito's Space Invaders but they did more modifications than just the title and company credit, it's a bit hard to decide it's "Taito (CompanyB license)". IIRC that's why I picked "Taito / CompanyB" for those.

The game is still clearly based on Taito's code, but the "Professional" mode makes it faster and has the invaders shoot three bullets instead of one. So I think the Taito / Jatre credit should work here.

**Pull request merged**